azure/compute/vm.go

```diff
@@ -72,15 +72,39 @@ func NewWindowsInstance(e azure.Environment, name, imageUrn, instanceType string
 		}
 	}
 
-	vm, publicIP, nw, err := newVMInstance(e, name, imageUrn, instanceType, windowsOsProfile)
+	securityGroup, err := network.NewNetworkSecurityGroup(e.Ctx, e.Namer.ResourceName(name, "security-group-rdp"), &network.NetworkSecurityGroupArgs{
```

Comment:
It should not be required while we already have the network bits in place. We need to understand what happens, rather than creating a NSG per VM.

Reply:
Can you elaborate how it should work? The network settings are the same for Windows and Linux which is strange as we want to expose RDP port for Windows only.

Reply:
The network setup is already created (vnet, subnet, nsg). The issue was that for some reason the subnet lost the default nsg setting. Fixed it and RDP/SSH works as expected now.